keras2onnx/_builtin.py

```diff
@@ -2187,6 +2187,33 @@ def convert_tf_strided_slice(scope, operator, container):
                                   operator.output_full_names,
                                   operator.inputs[0].full_name + '_identity')
 
+@converter_func(TYPES.TensorScatterUpdate)
+def convert_tf_tensor_scatter_update(scope, operator, container):
+    if operator.target_opset < 11:
+        raise ValueError("TensorScatterUpdate op is not supported for opset = " + str(operator.target_opset))
+    else:
+        oopb = OnnxOperatorBuilder(container, scope)
+        node = operator.raw_operator
+
+        indices = _cal_tensor_value(node.inputs[1])
+        indices_arr = np.array(indices)
+        indices_tensor = numpy_helper.from_array(indices_arr, operator.inputs[1].full_name + '_value')
+        container.add_initializer_from_tensor(indices_tensor)
+
+        updates_name = operator.inputs[2].full_name
+        updates = _cal_tensor_value(node.inputs[2])
+        if (updates != None):
+            updates_arr = np.array(updates)
+            updates_tensor = numpy_helper.from_array(updates_arr, operator.inputs[2].full_name + '_value')
+            container.add_initializer_from_tensor(updates_tensor)
+            updates_name = operator.inputs[2].full_name + '_value'
+
+        oopb.add_node_with_output('ScatterND',
```

Comment:
thanks for the contribution!
please add op_version=11 if any oopb.add_xxxx function was invoked.

Reply:
Thank you for the info. Added.

Reply:
Thank you for the info. Added.